posthog-core/src/index.ts

```diff
@@ -292,7 +294,8 @@ export abstract class PostHogCoreStateless {
         ...extraPayload,
       }),
     }
-    return this.fetchWithRetry(url, fetchOptions)
+    // Don't retry /decide API calls
+    return this.fetchWithRetry(url, fetchOptions, { retryCount: 0 }, this.featureFlagsRequestTimeout)
```

Comment:
If we have retrycount 0 then do we even want to call `fetchWithRetry` anymore? 🤔

Reply:
Good question, `fetchWithRetry` also handles timeouts, and didn't want to introduce yet another function just for timeouts. I could rename it to `fetchWithRetryAndTimeout` ?

Reply:
`fetchWithRetryAndOrTimeout` :D